test/parallel/test-promises-gc-before-handled.js

```diff
@@ -0,0 +1,24 @@
+// Flags: --expose-gc
+
+'use strict';
+const common = require('../common');
+
+const p = new Promise((res, rej) => {
+  consol.log('oops'); // eslint-disable-line no-undef
```

Comment:
No reason to cause an implicit reference error here - this can just be `throw new ReferenceError(...)`